docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -3,42 +3,88 @@ title: Set Up Code Builder
 lang: en
 ---
 
-## Install the Code Builder Managed Package
+Enable Code Builder to provide the permissions needed to install the Code Builder managed package in a Salesforce supported org.
 
-Salesforce Code Builder is a second-generation (2GP) managed package that a user with requisite permissions can install in any supported Salesforce org edition.
+### Required Editions
 
-1.  Click the install [link](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000HddeKUAR) to access the Code Builder managed package on AppExchange.
-2. Click **Get It Now**.
-3. Select the org in which you want to install Code Builder.
-4. Check terms and conditions and click **Confirm and Install**.
+**Available in**: Lightning Experience in Enterprise, Performance, Professional and Unlimited Editions.
 
+**Available in**: Government Cloud Plus as interoperable. Connecting Code Builder to Government Cloud Plus orgs can send data outside the authorization boundary. Contact your Salesforce account executive for more details.
 
-![Install Button](../../../images/install_button.png)
+**Inoperable in**: Government Cloud. Although Code Builder appears in the Setup menu, don’t enable or install it because it won’t operate properly.
 
-Grant access and click **Continue**.
+**Not available in**: EU Operating Zone. EU Operating zone is a special paid offering that provides an enhanced level of data residency commitment. Code Builder is supported in orgs in the EU that aren’t part of EU OZ, per standard product terms and conditions.
 
+## Enable and Install Code Builder
 
+To install Code Builder in a Professional Edition org, the org must have API access. If you attempt to install Code Builder in a Professional Edition org without API access, an installation error occurs. Contact your Account Executive to request the API add-on.
 
-![Grant Access](../../../images/grant_access.png)
+1. From Setup, enter Code Builder in the Quick Find box, then select Code Builder.
 
+2. Enable Code Builder, and then review and accept the license agreement, which allows you to install the package.You can disable the preference at any time. If Code Builder is already installed, disabling the preference prevents access to the Code Builder application.
 
+3. Click **Install Package**.
+   After the installer launches, you’re guided through the installation process to install the latest version of the Code Builder managed package. You can come back to this Setup page to reinstall or upgrade the package.
 
-Click **Done**.
+4. Select **Install for All Users**, and then click **Install**.
 
+5. Approve third-party access and click **Continue**.
+
+Code Builder Dashboard in now available in App Launcher. Be sure to assign the appropriate Code Builder permission sets to team member.
```

Comment:
@sbudhirajadoc - Small typo in the last sense - should be "...to team member**s**."